test/cases/010_platforms/000_qemu/001_run/test.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+# SUMMARY: Test running an image with qemu
+# LABELS:
+# AUTHOR: Dave Tucker <dt@docker.com>
+
+set -e
+
+# Source libraries. Uncomment if needed/defined
+#. "${RT_LIB}"
+. "${RT_PROJECT_ROOT}/_lib/lib.sh"
+
+IMAGE_NAME=test-qemu-build
+
+clean_up() {
+	# remove any files, containers, images etc
+	rm -rf "${LINUXKIT_TMPDIR:?}/${IMAGE_NAME:?}*" || true
+}
+
+trap clean_up EXIT
+
+# Test code goes here
+[ -f "${LINUXKIT_TMPDIR}/${IMAGE_NAME}-kernel" ] || exit 1
```

Comment:
Yes. I think I'll re-org the tests to fix this.

My new plan is to change `000_config` to `000_smoke` and to add some more smoke tests so that we're confident that a PR is good before proceeding to the next phase of testing.

The second phase would be run in a CI worker that has access to GCP etc... and it would just run the `platforms` tests.